wgpu-core/src/timestamp_normalization/mod.rs

```diff
@@ -68,6 +71,32 @@ pub enum TimestampNormalizerInitError {
     ComputePipeline(#[from] CreateComputePipelineError),
 }
 
+impl WebGpuError for TimestampNormalizerInitError {
```

Comment:
Since we don't need to `impl WebGpuError for RequestDeviceError`, we shouldn't need to implement it for `TimestampNormalizerInitError`, it would sidestep the mapping of `ErrorType::Validation` to `ErrorType::DeviceLost`.

Reply:
At some point we should probably revisit all variants of `RequestDeviceError`, errors returned by `IndirectValidation::new` and `TimestampNormalizerInitError` to make sure we satisfy the spec, but also surface some more information as to why device creation failed (for logging purposes).